src/Mono.Android/Test/Mono.Android-Tests.projitems

```diff
@@ -4,7 +4,9 @@
     <UnitTestApk Include="$(OutputPath)Mono.Android_Tests-Signed.apk">
       <Package>Mono.Android_Tests</Package>
       <InstrumentationType>xamarin.android.runtimetests.TestInstrumentation</InstrumentationType>
-      <ResultsPath>$(MSBuildThisFileDirectory)..\..\..\TestResult-Mono.Android_Tests.xml</ResultsPath>
+      <ResultsPath Condition=" '$(Configuration)' != 'Release' " >$(MSBuildThisFileDirectory)..\..\..\TestResult-Mono.Android_Tests.xml</ResultsPath>
```

Comment:
I want to avoid making the entire `$(ResultsPath)` value conditional, as it'll be too easy to overlook something when we inevitably need to update things later.

We could split the difference: in `tests/RunApkTests.targets`, before the `<Import/>`s, do:

```xml
<PropertyGroup>
  <_AotName Condition=" '$(AotAssemblies)' != '' ">-Aot</_AotName>
</PropertyGroup>
```

Then you should be able to use:

```xml
<ResultsPath>$(MSBuildThisFileDirectory)..\..\..\TestResult-Mono.Android_Tests-$(Configuration)$(_AotName).xml</ResultsPath>
```

> Plus I wasn't sure whether other parts (Jenkins, wrench?) use the `TestResult-Mono.Android_Tests.xml` filename

Jenkins does, but through file glob: at the [xamarin-android **Configuration** page](https://jenkins.mono-project.com/view/Xamarin.Android/job/xamarin-android/configure), in the **Publish xUnit test result report** section, is an **NUnit-2 (default) Pattern** textfield:

    xamarin-android/TestResult-*.xml

This is the file glob of files to look for which contain NUnit2 test result files.

Since this is a file glob, there should be no problem using a templatei-zed `%(UnitTestApk.ResultsPath)` value.